src/main/java/com/lgtm/easymoney/controllers/RequestController.java

```diff
@@ -48,22 +48,41 @@ public ResponseEntity<RequestRsp> createRequest(@Valid @RequestBody RequestReq r
   @Operation(summary = "Method for a user "
           + "to get all money requests sent (but not yet completed).")
   public ResponseEntity<RequestRsp> getRequests(@PathVariable(value = "uid") Long uid) {
-    // get all requests that are sent/received by user
-    // param: uid (from/to are the same for now) we may add filtering features later
-    // return: list of requests
-    return requestService.getRequestsByUser(userService.getUserById(uid));
+    return new ResponseEntity<>(requestService.getRequestsByUid(uid), HttpStatus.OK);
```

Comment:
I think here you need to check if it is success or not. If success, then the status is HttpStatus.OK; otherwise, it should be HttpStatus.BAD_REQUEST.

Reply:
BAD_REQUEST should be triggered by exceptions thrown